packages/components/src/Notification/InlineNotification/InlineNotification.module.scss

```diff
@@ -0,0 +1,9 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+.inlineNotification {
+  color: $color-purple-800;
+}
+
+.isReversed {
+  color: $color-yellow-600;
+}
```

Comment:
Do these need to be changed?

Reply:
I just realized these came from plop and don't need to be here! Good catch  👁️🦅